### PR TITLE
Update tools.yml

### PR DESCRIPTION
**Pull request by patrickpseguin**
adding ESDC Standard Security Tools by Security Testing Type

### CODE REVIEW

File: _data/tools.yml [hunk 1]
```diff
@@ -100,3 +100,25 @@ development:
       - name: Selenium
         documentation: https://www.seleniumhq.org/
     
+- focus: ESDC Standard Security Tools by Security Testing Type
```
Comment (s-laugh) [inline _data/tools.yml]: We'll need to fix the tabbing so it loads properly, and the `by Security Testing Type` won't be needed as the `scope` will be appended to the title.
```suggestion
  - focus: ESDC Standard Security Tools
```

File: _data/tools.yml [hunk 2]
```diff
@@ -100,3 +100,25 @@ development:
...
+    scope: SAST (Legacy & Cloud)
+    tools:
+      - name: Microfocus Fortify Static Code Analyzer
+        documentation: https://www.microfocus.com/en-us/products/static-code-analysis-sast/overview
+    scope: DAST (Legacy & Cloud)
```
Comment (s-laugh) [inline _data/tools.yml]: We'll also need to add the focus in each time... (and the space is nice for separation 😃)
```suggestion

  - focus: ESDC Standard Security Tools
    scope: DAST (Legacy & Cloud)
```

File: _data/tools.yml [hunk 3]
```diff
@@ -100,3 +100,25 @@ development:
...
+    tools:
+      - name: Microfocus Fortify WebInspect
+        documentation: https://www.microfocus.com/en-us/products/webinspect-dynamic-analysis-dast/overview
+    scope: SCA (Legacy & Cloud)
```
Comment (s-laugh) [inline _data/tools.yml]: Same thing here
```suggestion

  - focus: ESDC Standard Security Tools
    scope: SCA (Legacy & Cloud)
```

File: _data/tools.yml [hunk 4]
```diff
@@ -100,3 +100,25 @@ development:
...
+    tools:
+      - name: Sonatype Nexus Lifecycle
+        documentation: https://www.sonatype.com/product-nexus-lifecycle
+    scope: AVM (Cloud only)
+    tools:
+      - name: Denim Group Threadfix
+        documentation: https://threadfix.it/
+    scope: Container Security (Cloud only)
```
Comment (s-laugh) [inline _data/tools.yml]:
```suggestion

  - focus: ESDC Standard Security Tools
    scope: Container Security (Cloud only)
```

File: _data/tools.yml [hunk 5]
```diff
@@ -100,3 +100,25 @@ development:
...
+        documentation: https://www.microfocus.com/en-us/products/static-code-analysis-sast/overview
+    scope: DAST (Legacy & Cloud)
+    tools:
+      - name: Microfocus Fortify WebInspect
+        documentation: https://www.microfocus.com/en-us/products/webinspect-dynamic-analysis-dast/overview
+    scope: SCA (Legacy & Cloud)
+    tools:
+      - name: Sonatype Nexus Lifecycle
+        documentation: https://www.sonatype.com/product-nexus-lifecycle
+    scope: AVM (Cloud only)
```
Comment (s-laugh) [inline _data/tools.yml]:
```suggestion

  - focus: ESDC Standard Security Tools
    scope: AVM (Cloud only)
```

File: _data/tools.yml (hunk 4 above)
Comment (s-laugh) [inline _data/tools.yml]:
```suggestion

  - focus: ESDC Standard Security Tools
    scope: Container Security (Cloud only)
```

**Comment by patrickpseguin** [comment]
Formatting changes applied.

**Pull request merged**